### PR TITLE
Refactor: Rename AddressBook to ManageHR

### LINKED ISSUE #38: Modify file/class/method/variable names to conform with ManageHR instead of AddressBook
Convert all file file/class/method/variable to conform with ManageHR

### PR DESCRIPTION
**Pull request by kenvynKwek**
Renamed instances (class/methods/objects) of `AddressBook` to `ManageHr`
- no change in functionality
- all test cases passed

### CODE REVIEW

**Comment by SungMatt** [comment]
You can use git add -f <filename> if you wanna bypass .gitignore!

File: src/main/java/seedu/address/storage/ManageHrStorage.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataLoadingException;
+import seedu.address.model.ManageHr;
```
Comment (SungMatt) [inline src/main/java/seedu/address/storage/ManageHrStorage.java:8]: Is this line a required import?

Comment (kenvynKwek) [reply src/main/java/seedu/address/storage/ManageHrStorage.java:8]: yes it's utilised in line 12's (of this same file) @link tag for javadocs. this was also the case previously for AddressBook3

**Review by SungMatt — APPROVED** [review]
I think this looks good to go. Anyone have any other opinions? Otherwise I'll just merge this in about 12 hours from now.

File: src/main/java/seedu/address/storage/ManageHrStorage.java (hunk 1 above)
Comment (SungMatt) [reply src/main/java/seedu/address/storage/ManageHrStorage.java:8]: Ah -- was confused as to why AddressBookStorage.java didn't have this line. Sure!

**Comment by SungMatt** [comment]
Thanks Kenvyn!

**Pull request merged**